src/test/app/claims/models/claim.ts

```diff
@@ -148,4 +154,73 @@ describe('Claim', () => {
       expect(claim.defendantOffer).to.be.eq(undefined)
     })
   })
+
+  describe('claimStatus', () => {
+
+    it('should return ELIGIBLE_FOR_CCJ if defendant has not responded before the deadline', () => {
+      const claim = buildClaim()
```

Comment:
You can use a `beforeEach` block to build the claim. This way you can avoid doing the same initialisation in each test. For example:
```
describe('claimStatus', () => {
  let claim

  beforeEach(() => {
    claim = buildClaim()
  })
```